metricflow/dataflow/builder/dataflow_plan_builder.py

```diff
@@ -250,6 +251,9 @@ def _build_aggregated_conversion_node(
         # Due to other outstanding issues with conversion metric filters, we disable predicate
         # pushdown for any filter parameter set that is not part of the original time range constraint
         # implementation.
+        default_granularity = ExpandedTimeGranularity.from_time_granularity(
```

Comment:
nit: can we move this so that the comment above will still be next to the code it's referencing?